image/qcow2/qcow2.go

```diff
@@ -536,6 +536,7 @@ type Qcow2 struct {
 	errUnreadable       error
 	clusterSize         int
 	l1Table             []l1TableEntry
+	l2TablesCache       map[l1TableEntry][]l2TableEntry
```

Comment:
Needs a mutex

Reply:
Do we support access from multiple threads?

Reply:
Current version uses a thread safe lru cache.